article.tex

```diff
@@ -75,34 +75,34 @@
 
 
 \section[Introduction]{Introduction}
-Before each statistical analysis, data cleaning is often required to ensure a good data quality.
-Data cleaning is the process of detecting errors and inconsistencies in data sets \citep{Chapman_2005}.
+Before each statistical analysis, data cleaning is often required to ensure good data quality.
+Data cleaning is the process of detecting errors and inconsistencies in datasets \citep{Chapman_2005}.
```

Comment:
Not sure about this.
http://english.stackexchange.com/questions/2120/which-is-correct-dataset-or-data-set
http://forum.wordreference.com/threads/dataset-vs-data-set.1118035/

data set is the more general term, while dataset i a more specific one (though the differences are neglectable).